Sources/CXTestUtility/Inconsistent/VersioningExpectation.swift

```diff
@@ -26,6 +27,16 @@ public extension Expectation {
     }
 }
 
+public func context(minimalVersion: XcodeVersion, closure: () -> Void) {
+    let description = "tests requires minimal system version shipping with Xcode \(minimalVersion.version)"
+    #if canImport(Darwin)
+    let enabled = ProcessInfo.processInfo.operatingSystemSemanticVersion > minimalVersion.systemVersion
+    #else
+    let enabled = true
+    #endif
+    context(description, flags: ["pending": !enabled], closure: closure)
+}
```

Comment:
expressive!

Reply:
It prevents CombineX tests from running on older system version. I've fixed it, could you please review again?